packages/core/src/components/TextField/Styled/TextField.styled.tsx

```diff
@@ -48,6 +57,7 @@ export const Input = styled('input')<StyledProps>`
     text-overflow: ellipsis;
     resize: none;
     z-index: 1;
+    padding: 0;
```

Comment:
Why we removed the 0 padding ?

Reply:
When I don't set the padding to 0 here, there is 2px of padding on each side of the input div that's set by the browser. Removing the padding here so that the input text is aligned with the label text.

Reply:
<img width="346" alt="Screen Shot 2020-10-29 at 9 37 38 AM" src="https://user-images.githubusercontent.com/43207325/97610435-3b6de980-19eb-11eb-858f-6da552f8a0a3.png">

Reply:
<img width="353" alt="Screen Shot 2020-10-29 at 9 37 29 AM" src="https://user-images.githubusercontent.com/43207325/97610457-432d8e00-19eb-11eb-980b-8a1d03c063e9.png">